scripts/Makefile

```diff
@@ -181,12 +183,16 @@ ifeq ($(usingLinux),1)
   endif
 
   linkerFlags += -lm -lrt -ldl
+  soNameFlag = -Wl,-soname,libocca.so
+  soExt = so
 
 else ifeq ($(usingMacOS),1)
 
 	OCCA_OS := OCCA_MACOS_OS
   flags += -Wno-deprecated-declarations
   linkerFlags += -framework accelerate -framework CoreServices
+  soNameFlag = -Wl,-install_name,@rpath/libocca.dylib
```

Comment:
This could be
```sh
 -Wl,-install_name,$(PREFIX)/lib/libocca.dylib
```

as well. Not sure what to go for.

Reply:
We can merge this first and add `$(PREFIX)` in another commit if it becomes an issue